src/main/java/com/fasttime/domain/resume/controller/ResumeController.java

```diff
@@ -0,0 +1,82 @@
+package com.fasttime.domain.resume.controller;
+
+import com.fasttime.domain.resume.dto.ResumeDeleteServiceRequest;
+import com.fasttime.domain.resume.dto.ResumeRequestDto;
+import com.fasttime.domain.resume.dto.ResumeResponseDto;
+import com.fasttime.domain.resume.dto.ResumeUpdateRequest;
+import com.fasttime.domain.resume.dto.ResumeUpdateServiceRequest;
+import com.fasttime.domain.resume.service.ResumeService;
+import com.fasttime.global.util.ResponseDTO;
+import com.fasttime.global.util.SecurityUtil;
+import jakarta.validation.Valid;
+import java.time.LocalDateTime;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v2/resume")
+public class ResumeController {
+
+    private final SecurityUtil securityUtil;
+    private final ResumeService resumeService;
+
+    // Resume create
+    @PostMapping("/create")
+    public ResponseEntity<ResponseDTO<ResumeResponseDto>> creatResume(
```

Comment:
오타가 있는 것 같아요~ creatResume -> createResume
```suggestion
    public ResponseEntity<ResponseDTO<ResumeResponseDto>> createResume(
```